mne/gui/__init__.py

```diff
@@ -247,6 +247,44 @@ def locate_ieeg(info, trans, aligned_ct, subject=None, subjects_dir=None,
     return gui
 
 
+@verbose
+def view_stc(stc, subject=None, subjects_dir=None, src=None,
+             show=True, block=False, verbose=None):
+    """Locate intracranial electrode contacts.
```

Comment:
This looks like it needs an update.

Reply:
This is the work in progress GUI, it's almost ready to push a version without all these errors just squashing a few bugs! It's much more extensive than this but only on my local branch. The GUI isn't ready for review yet (it will be very soon though!), when I said above that the PR could be reviewed, I meant just the source time course-generating part, sorry if that was unclear.